### PR TITLE
Experimental Changes: 2

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

Long-awaited sequel to the first experimental PR. This will include unorthodox methods of improving the game in one way or the other.

- Players are unable to respawn when ordeals are still running. **This is only active above 30 pop (temporary).**
- If all agents die during an ordeal - the round will start a countdown (2 minutes) until an automatic restart. If a new player joins or all ordeals end - the countdown is stopped. **This is only active above 30 pop (temporary).**
- Ordeal random delays are now between -1 and 1, instead of 0 and 2.
- Abnormality queue is less random: When you have two or more threat levels available (i.e. HE and WAW), the queue will try to ensure that you don't get only one threat level (i.e. only HEs).
- If an ordeal of a high level ends before you have appropriate abnormalities for the next one (for example, dusk when you don't have ALEPHs) - a maximum of 4 abnormalities will be spawned in quick succession, until you have the expected theat level in the queue.
- Each living agent contributes exactly 1 work to max qliphoth counter, instead of 0.8.
- Ghosts and officers (EO, RO) can see anyone's attributes by examining (a link "Check Attributes" will be beneath equipment description).

## Why It's Good For The Game

- Prevents the human wave tactic that extends a lost round for another hour. Sometimes a loss is a loss, and we were missing such a condition. **In theory a single player can hold the round hostage, but to that I have to say that restart vote exists and is always available to players. If someone is maliciously ignoring ordeals - this is a bannable offense as per rule 16.**
- A long-awaited loss condition. This prevents rounds from going on forever when all is clearly lost.
- Less RNG in the game. Midnight now doesn't take godawful amount of time to start.
- No longer shall you get full queue of HEs when you need WAWs, or WAWs when you need ALEPHs.
- Helps with the situations where peopl were working too fast and didn't have WAWs for a dusk, or ALEPHs for a midnight.
- This simply makes more sense - each agent will get at least one work in, regardless of pop.
- A simple neat detail for ghosts, allowing them to see how good everyone is doing. For EO - this allows them to see who can wield higher-level EGO at a glance.

### CODE REVIEW

**Review by Kitsunemitsu — DISMISSED** [review]
I'm not approving this for merge on points 3 and 4, see my discord message for more.

**Comment by EgorDinamit** [comment]
> I'm not approving this for merge on points 3 and 4, see my discord message for more.

As is customary with ***experimental changes*** - everything will most likely be changed by the time it should be merged.
I personally put DNM label for a good reason.

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -699,6 +699,13 @@
 		to_chat(usr, "<span class='boldnotice'>You must be dead to use this!</span>")
 		return
 
+	if(length(GLOB.clients > 5) && LAZYLEN(SSlobotomy_corp.current_ordeals))
```
Comment (LanceSmites328) [inline code/modules/mob/mob.dm:702]:
```suggestion
	if((length(GLOB.clients) > 5) && LAZYLEN(SSlobotomy_corp.current_ordeals))
```

**Review by LanceSmites328 — DISMISSED** [review]

**Comment by LanceSmites328** [comment]
Overall changes 1 and 2 are good. Changes 3 and 4 just aren't working.

**Comment by EgorDinamit** [comment]
PR's body updated to represent new changes.

**Comment by Kitsunemitsu** [comment]
Update comment again, if you would.

**Comment by LanceSmites328** [comment]
Code looks good, awaiting the discussion before going ahead with it, however.

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**